### DISCUSSION TITLE
Dart client:  can I generate models (classes) that are mutable?

### DISCUSSION THREAD

**Discussion by dgaedcke** [prisma/prisma#23273] [Q&A]
### Question

or alternatively, can we generate a "builder" class for each of my immutable models?   I'd like to use these models to collect data along the way (during user form entry) which prevents me from constructing them in one-shot!

prisma -v:  10.2.4

**Comment by ludralph** [comment]
Hi @dgaedcke 👋 

Thank you for raising this question.

Can you please raise this question at the Prisma Client Dart [Githbub Discussions](https://github.com/medz/prisma-dart/discussions)? That is the correct place to receive an answer to your specific query.
